lib/matplotlib/contour.py

```diff
@@ -887,11 +887,12 @@ def __init__(self, ax, *args,
         self.set_cmap(cmap)
         if norm is not None:
             self.set_norm(norm)
-        if vmin is not None:
-            self.norm.vmin = vmin
-        if vmax is not None:
-            self.norm.vmax = vmax
-        self._process_colors()
+        with self.norm.callbacks.blocked(signal="changed"):
```

Comment:
`blocked` doesn't process any pending signals after the context; do you need to trigger a single signal after?